ML-Frameworks/pytorch-aarch64/CHANGELOG.md

```diff
@@ -35,7 +33,7 @@ https://github.com/ARM-software/Tool-Solutions/tree/r25.03.1
 - Move torch packages to latest stable to fix build error
   - torchaudio==2.6.0.dev20250305 -> 2.6.0
   - torchdata~=0.7.1 -> 0.11.0
-  - torchvision~=0.22.0.dev20250305 -> 0.21.0
+  - torchvision~=0.22.0.dev20250305 -> ~=0.22.0.dev20250403
```

Comment:
This has been released right? We can't change it. Is this bullet meant to be above?

Reply:
I've updated the list so the torch vision update isn't lumped in with the others any more.